internal/files/search_test.go

```diff
@@ -32,165 +32,212 @@ func TestSearchFiles(t *testing.T) {
 		{
 			filePath: "../../example/src/go/SDK_V2/sample.go",
 			flags: []flag{
-				{name: "btnColor", lineNumber: 31, codeLineHighlight: 6},
-				{name: "btnColor", lineNumber: 32, codeLineHighlight: 6},
-				{name: "btnColor", lineNumber: 33, codeLineHighlight: 6},
-				{name: "btnColor", lineNumber: 34, codeLineHighlight: 6},
-				{name: "btnColor", lineNumber: 35, codeLineHighlight: 6},
+				{name: "btnColor", lineNumber: 36, codeLineHighlight: 6},
+				{name: "btnSize", lineNumber: 37, codeLineHighlight: 6},
+				{name: "showBtn", lineNumber: 38, codeLineHighlight: 6},
 			},
 		},
-		{
+		/* 		{
```

Comment:
remove is better than let a useless comment I think

Reply:
this is supposed to be a test, I removed the comments and fixed the test